.travis.yml

```
@@ -23,7 +23,8 @@ install:
     - enact link
 script:
     - npm run lint -- -- --report-unused-disable-directives --max-warnings 0 .
-    - npm run test
+    - npm run test -- --runInBand --coverage
```

Could you please add echo statement as other repositories?
```suggestion
    - echo -e "\x1b\x5b35;1m*** Starting tests...\x1b\x5b0m"
     - echo -e "\x1b\x5b35;1m*** Tests complete\x1b\x5b0m"
```
and also 
```suggestion
    - echo -e "\x1b\x5b35;1m*** Starting eslint...\x1b\x5b0m"
    - echo -e "\x1b\x5b35;1m*** eslint complete\x1b\x5b0m"
```